src/ui/react/hooks.ts

```diff
@@ -19,7 +19,9 @@ export interface MassUpdater<Domain, K extends keyof Domain>
 type UpdaterInternal<T> = (options: {}, delta: (prev: T) => T) => void;
 export type Updater<T> = UpdaterInternal<T> & {
   (delta: (prev: T) => T): void;
-};
+} & (T extends {}
+    ? { <K extends keyof T & string>(field: K, value: T[K]): void }
```

Comment:
not a critique just a question for understanding: `keyof T & string` -- Is this saying "all string keys of T"? Because keys could technically also be a Symbol?

Reply:
Exactly. I wanted to make sure we don’t support symbol keys here.